src/modules/material-requests/types/index.ts

```diff
@@ -16,6 +16,31 @@ export interface MaterialRequest {
 	updatedAt: string;
 }
 
+export interface MaterialRequestDetails {
```

Comment:
In case there's much overlap with the `MaterialRequest` type, you could extend the new type:

```
export interface MaterialRequestDetails extends MaterialRequest {
    // add extra properties here
}
```